cmd/akash/logs.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/ovrclk/akash/cmd/akash/session"
+	"github.com/ovrclk/akash/keys"
+	"github.com/ovrclk/akash/types"
+	"github.com/spf13/cobra"
+)
+
+func logsCommand() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "logs <lease>",
+		Short: "service logs",
+		Args:  cobra.ExactArgs(2),
+		RunE:  session.WithSession(session.RequireNode(logs)),
+	}
+
+	session.AddFlagNode(cmd, cmd.PersistentFlags())
+	cmd.Flags().Int64P("lines", "l", 10, "Number of lines from the end of the logs to show per service")
+	cmd.Flags().BoolP("follow", "f", false, "Follow the log stream of the service")
+
+	return cmd
+}
+
+func logs(session session.Session, cmd *cobra.Command, args []string) error {
+	serviceName := args[0]
+	leasePath := args[1]
+	lease, err := keys.ParseLeasePath(leasePath)
+	if err != nil {
+		return err
+	}
+	provider, err := session.QueryClient().Provider(session.Ctx(), lease.Provider)
+	if err != nil {
+		return err
+	}
+	tailLines, err := cmd.Flags().GetInt64("lines")
+	if err != nil {
+		return err
+	}
+	follow, err := cmd.Flags().GetBool("follow")
+	if err != nil {
+		return err
+	}
+	options := types.LogOptions{
+		TailLines: tailLines,
+		Follow:    follow,
+	}
+	b, err := json.Marshal(options)
+	if err != nil {
+		return err
+	}
+	url := provider.HostURI + "/logs/" + leasePath + "/" + serviceName
+	body, err := stream(session.Ctx(), url, b)
```

Comment:
To have this shut down gracefully, wrap it in a `common.RunForever()`.  Though as I type this I realize that maybe we should wrap everything in that.